src/state/reducers/month-state/employee-info.reducer.ts

```diff
@@ -40,10 +48,16 @@ export function employeeInfoReducerF(name: string) {
         delete state.time[workerName];
         delete state.type[workerName];
         delete state.contractType?.[workerName];
+        delete state.employmentTime?.[workerName];
+        delete state.employmentTimeOther?.[workerName];
+        delete state.civilTime?.[workerName];
```

Comment:
Generalnie, to nie jest dobrze, ponieważ teraz w naszym stanie pojawia się redundancja. 
Inaczej mówiąc, jeśli my chcemy zmienić etat pracownika, to teraz potrzeba będzie zmieniać to w 4 miejscach:
`time, employmentTime, employmenyTimeOther, civilTime`.

Co w przypadku jeśli my zapomnimy o tym, żeby gdzieś to zrobić? 
Wtedy pojawią się anomalii związane z tym, że w jednym komponencie czas pracownika będzie wynosić jedną wartość, a w jednym inną. 

Zamiast dodawać te trzy nowe pola, ja bym przechowywał wszystkie niezbędne informacje w polu `time`.

Żeby wyciągnąć z tego pola informację o employmentTime, employmentTimeOther i civilTime, można by było napisać funkcje pomocnicze, które mogłyby to przeliczać.